x-pack/legacy/plugins/lens/public/indexpattern_plugin/field_icon.tsx

```diff
@@ -17,6 +17,7 @@ function getIconForDataType(dataType: string) {
   const icons: Partial<Record<string, UnwrapArray<typeof ICON_TYPES>>> = {
     boolean: 'invert',
     date: 'calendar',
+    ip: 'compute',
```

Comment:
+1 for a specific "IP" icon.  I like `storage` second best till that's ready.

Reply:
note that with #46485 discover will use the same icons (for now in black, but coloring is already built in), there are a few more here, but I think it would make sense to have 1 shared component just for this icons. A specific IP icon would be welcome, but also the storage one is fine